### PR TITLE
Remove some console warnings (DataTable)

### PR DESCRIPTION
**Pull request by eadams17**
## WHAT
remove some console warnings being caused by the `DataTable` component

## WHY
we don't want this noise in the console

## HOW
add some missing keys for iterative elements. I also tweaked a cell element in the activity packs so that it would be accessible 

note: most of these file changes are snapshot updates that can be ignored

### Screenshots
(If applicable. Also, please censor any sensitive data)

### Notion Card Links
(Please provide links to any relevant Notion card(s) relevant to this PR.)
https://www.notion.so/quill/Audit-and-inventory-of-front-end-console-warnings-20c4f7d53f07422b9449960f9d3e359c

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? | yes
Have you deployed to Staging? | yes
Self-Review: Have you done an initial self-review of the code below on Github? | yes
Spec Review: Have you reviewed the spec and ensured this meets requirements and/or matches design mockups? | yes

### CODE REVIEW

File: services/QuillLMS/client/app/bundles/Shared/components/shared/tooltip.tsx [hunk 1]
```diff
@@ -93,7 +93,7 @@ class Tooltip extends React.Component<TooltipProps, { clickedFromMobile: boolean
     const { tooltipVisible, } = this.state
     const tabIndex = isTabbable ? 0 : null;
 
-    const triggerClass = `quill-tooltip-trigger ${tooltipVisible ? 'active' : ''}`
+    const triggerClass = `quill-tooltip-trigger  ${tooltipVisible ? 'active' : ''}`
```
Comment (anathomical) [inline services/QuillLMS/client/app/bundles/Shared/components/shared/tooltip.tsx]: Looks like an extra space snuck in here that's not necessary?

**Review by anathomical — APPROVED** [review]
Looks good.  One call out of a possible extraneous space, but it's not actually very important.

File: services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/custom_activity_pack/__tests__/__snapshots__/mobile_filter_menu.test.tsx.snap [hunk 2]
```diff
@@ -29591,7 +29591,7 @@ Combine simple sentences to create 5 new compound or complex sentences that incl
                     >
                       <span
                         aria-hidden={false}
-                        className="quill-tooltip-trigger "
+                        className="quill-tooltip-trigger  "
```
Comment (happythenewsad) [inline services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/custom_activity_pack/__tests__/__snapshots__/mobile_filter_menu.test.tsx.snap]: Do you know why this is happening?

**Review by happythenewsad — APPROVED** [review]
Nice work! 

Please comment on the white space diff though - IDE artifact?

File: services/QuillLMS/client/app/bundles/Shared/components/shared/tooltip.tsx (hunk 1 above)
Comment (eadams17) [reply services/QuillLMS/client/app/bundles/Shared/components/shared/tooltip.tsx]: ahh yeah, please disregard! I was testing something and must have accidentally left a space

File: services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/custom_activity_pack/__tests__/__snapshots__/mobile_filter_menu.test.tsx.snap (hunk 2 above)
Comment (eadams17) [reply services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/custom_activity_pack/__tests__/__snapshots__/mobile_filter_menu.test.tsx.snap]: see my response to Thomas

**Pull request merged**